tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -1245,3 +1245,43 @@ def _phi(m):
         return np.sum(np.log(C)) / (N - m + 1.0)
 
     return np.abs(_phi(m) - _phi(m + 1))
+
+@set_property("fctype", "aggregate_with_parameters")
+def max_fixed_point(x, r, m):
+    """
+    Largest fixed point of dynamics estimated from polynomial, which has been fitted to 
+    the deterministic dynamics of Langevin model as described by
+
+        Friedrich et al. (2000): Physics Letters A 271, p. 217-222
+        *Extracting model equations from experimental data*
+
+    For short time-series this method is highly dependent on the parameters.
```

Comment:
I think I unterstand the feature. It fits a polynomial over quantile-binned changes of the time series. It would be nice to have a formula of that in the docstring